lms/djangoapps/courseware/module_render.py

```diff
@@ -182,6 +183,7 @@ def toc_for_course(request, course, active_chapter, active_section, field_data_c
                                      })
             toc_chapters.append({
                 'display_name': chapter.display_name_with_default,
+                'display_id': name_without_spaces,
```

Comment:
@ahsan-ul-haq I would call the variable ``display_id`` since it isn't a name without spaces anymore.